code/go/0chain.net/smartcontract/dbs/event/process_test.go

```diff
@@ -35,6 +37,7 @@ func TestAddEvents(t *testing.T) {
 		},
 	})
 	errObj := Error{}
+	time.Sleep(100 * time.Millisecond)
```

Comment:
?

Reply:
This is required because now that we use channels the goroutine needs to be scheduled after adding events